gitless/cli/gl_fetch.py

```diff
@@ -23,4 +23,4 @@ def parser(subparsers, _):
 def main(args, repo):
     core.git_wrap('fetch', *args.fetch_args)
```

Comment:
add `p = ` before `core.git_wrap(....` and you should be good to go.

Reply:
Done. Maybe I'll need more coffee :D